sct-commons/src/test/java/org/lfenergy/compas/scl2007b4/model/NamespaceConfigurationTest.java

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: 2023 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.scl2007b4.model;
+
+import org.junit.jupiter.api.Test;
+import org.lfenergy.compas.sct.commons.scl.PrivateService;
+import org.lfenergy.compas.sct.commons.testhelpers.MarshallerWrapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class NamespaceConfigurationTest {
+
+    @Test
+    void marshalling_SCL_and_Compas_Privates_should_set_correct_prefix() {
+        // Given
+        SCL scl = createValidScl();
+        TPrivate aCompasPrivate = PrivateService.createPrivate(TCompasSclFileType.SCD);
+        scl.getPrivate().add(aCompasPrivate);
+        // When
+        String result = MarshallerWrapper.marshall(scl);
+        // Then
+        System.out.println(result);
+        assertThat(result)
+                .contains("<SCL") // http://www.iec.ch/61850/2003/SC namespace should not have any prefix
```

Comment:
your test is not really located on what you are saying in comment...

You could do something more accurate;

```java
// Then
        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
        factory.setNamespaceAware(true);
        DocumentBuilder documentBuilder = factory.newDocumentBuilder();

        InputSource inputSource = new InputSource(new StringReader(xml));
        Document document = documentBuilder.parse(inputSource);

        // http://www.iec.ch/61850/2003/SC namespace should not have any prefix
        assertThat(document.getDocumentElement().getAttributes().item(3))
                .extracting(Node::getNodeName, Node::getNodeValue)
                .containsExactly("xmlns", "http://www.iec.ch/61850/2003/SCL");

        // https://www.lfenergy.org/compas/extension/v1 namespace should have "compas" prefix
        assertThat(document.getDocumentElement().getAttributes().item(4))
                .extracting(Node::getNodeName, Node::getNodeValue)
                .containsExactly("xmlns:compas", "https://www.lfenergy.org/compas/extension/v1");
```

Reply:
or Something "String-related" but located on the xlmns

Reply:
I prefer to test output String directly, rather than convert it into a Java object again to avoid potential side effects in the conversion.
I suggest 
```java
assertThat(result)
        .containsPattern("(?s)<SCL[^>]* xmlns=\"http://www\\.iec\\.ch/61850/2003/SCL\"")
        .containsPattern("(?s)<SCL[^>]* xmlns:compas=\"https://www\\.lfenergy\\.org/compas/extension/v1\"");
```